build.gradle

```diff
@@ -252,8 +252,8 @@ subprojects {
     }
 
     plugins.withId("java") {
-        sourceCompatibility = 1.7
-        targetCompatibility = 1.7
+        sourceCompatibility = 1.8
+        targetCompatibility = 1.8
```

Comment:
Also update README.md and build.gradle files in example/ in this PR?

Reply:
README change needs to be part of the release (or maybe after, because we don't have a good process?). I've updated the examples.